Dockerfile

```diff
@@ -2,6 +2,8 @@
 FROM harbor.k8s.libraries.psu.edu/library/ruby-2.7.6-node-12:20230116 as base
```

Comment:
```suggestion
FROM harbor.k8s.libraries.psu.edu/library/ruby-2.7.6-node-12:20230126 as base
```

Reply:
@ajkiessl I have re-ran the builds for ruby-base, and believe that they should now have the new gpg key